exercises/grade-school/grade-school.ts

```diff
@@ -0,0 +1,5 @@
+class GradeSchool {
+    // Your code goes here
```

Comment:
I think it'd be good to add methods named `addStudent`, `studentRoster`, `studentInGrade`